src/graphql/schemas/inputs/sortOptions.ts

```diff
@@ -8,7 +8,7 @@ import type { AttestationSchema } from "../typeDefs/attestationSchemaTypeDefs.js
 import type { Fraction } from "../typeDefs/fractionTypeDefs.js";
 import { Order } from "../typeDefs/orderTypeDefs.js";
 import { Sale } from "../typeDefs/salesTypeDefs.js";
-import { Collection } from "../typeDefs/collectionTypeDefs.js";
+import { Hyperboard } from "../typeDefs/hyperboardTypeDefs.js";
 
 export type SortOptions<T extends object> = {
```

Comment:
Not sure where to leave this, but the sort options for the `User` are missing in the playground

Reply:
Will be in a different PR, and not necessary as of now